src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+            [0, building.people_1 * building.heat_loss_people_1][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_1, building.end_time_people_1)
+    ]
+    return people_at_times
+
+
+def internal_load_people_2(building: Clinic):
+    """
+    Computes the internal heat load of the patients and accompainers in kWh for a building.
+
+    """
+    people_at_times = [
+        [
+            [0, building.people_2 * building.heat_loss_people_2][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_2, building.end_time_people_2)
+    ]
+    return people_at_times
+
+
+def calculate_internal_load_people(building: Clinic):
+    """
+    Computes the internal heat load of the total people in kWh for a building.
+
+    """
+    p1 = internal_load_people_1(MY_CLINIC)
+    p2 = internal_load_people_2(MY_CLINIC)
+
+    print(p1[i] + p2[i] for i in range(len(p1[0])))
+
+
+calculate_internal_load_people(MY_CLINIC)
+
+
+def internal_load_lighting(building: Clinic):
+    """
+    Computes the internal heat load of the lamps in kWh for a building.
+
+    """
+    lamps_at_times = [
+        [
+            [0, building.lamps_internal * building.wattage_lamps][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_lamps, building.end_time_lamps)
+    ]
+    return lamps_at_times
+
+
+def equipment_load(building: Clinic):
+    """
+    Computes the internal heat load of the fridge in kWh for a building.
+
+    """
+
+    return building.fridge_wattage * building.time_fridge / 1000
+
+
+def infiltration_load(building: Clinic, temperature):
+    """
+    Computes the infiltration load in kWh for a building.
+
+    """
+    infiltration_at_times = [
+        [
+            [
+                0,
+                building.changes
+                * building.volume_cold_store
+                * building.energy_new_air
+                * (temperature - building.inside_ideal_temperature),
+            ][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(
+            building.start_time_infiltration, building.end_time_infiltration
+        )
+    ]
+    return infiltration_at_times
+
+
+# infiltration_load(MY_CLINIC)
+
+
+def calculate_cooling_load(building: Clinic, temperature):
+    """
+    Computes the total cooling load in kWh for a building.
+
+    """
+
+    q1 = calculate_transmission_load(MY_CLINIC, temperature)
+    q2 = calculate_internal_load_people(
+        MY_CLINIC,
+    )
+    q3 = internal_load_lighting(MY_CLINIC)
+    q4 = equipment_load(MY_CLINIC)
+    q5 = infiltration_load(MY_CLINIC, temperature)
+
+    print(q1, type(q1))
+    print(q2, type(q2))
+    print(q3, type(q3))
+    print(q4, type(q4))
+    print(q5, type(q5))
+    return (q1 + q2[i] + q3[i] + q4 + q5[i] for i in range(len(q2[0])))
```

Comment:
### ⌨️ Type mismatch?
Is there a mismatch between the types of `q1` and `q4` when compared with `q2`, `q3` and `q5`? It looks like you're just taking values from these lists, but the others are static, i.e., floats..? And this iterating through is going to return a `Generator` instance, rather than what I think you want, which is a sum?

Reply:
Yes, with q1 and q2 not depending on the hours of the day I was not sure on how to deal with them in this situation.. What is the best way to make it a sum of all the singular hourly values of every function?